README.md

```
@@ -79,6 +79,41 @@ cd my-app
 meteor
 ```
 
+# 💖 Sponsors
+
+Meteor is made possible by our amazing sponsors. Support the project and get your logo here by visiting our [Sponsorship page](https://www.meteor.com/sponsorship).
+
+### 🌟 Supernova Sponsors
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)